client/lib/reader-post-flux-adapter/index.js

```diff
@@ -68,27 +70,41 @@ const fluxPostAdapter = Component => {
 
 		render() {
 			const { post, posts } = this.state;
-			if ( ! post && ! posts ) {
-				return null;
-			}
-			return <Component { ...{ ...this.props, post, posts } } />;
+			return (
+				<div>
```

Comment:
a div just for query components make me 😢 

not sure what else to do though. hrm.